### ISSUE TITLE
Changes to docs fomatting

### ISSUE THREAD

**Issue by MrRedstone058** [pygame-community/pygame-ce#3130]
## The Issue:

> Currently, there is a disparity between the oncoming experimental Circle documentation and the old Rect/Surface/math documentation as shown below.

__Geometry docs__
![Screenshot_20240929_111648_Chrome](https://github.com/user-attachments/assets/c7bff155-3baa-4157-906d-080c910fc668)
![Screenshot_20240929_112034_Chrome](https://github.com/user-attachments/assets/d6f9e22b-1bc3-4084-8161-52cdc2c3c920)

__Rect docs__
![Screenshot_20240929_111713_Chrome](https://github.com/user-attachments/assets/cfea80d4-745d-453c-88ab-00f87b645cdb)

__Surface docs__
![Screenshot_20240929_111809_Chrome](https://github.com/user-attachments/assets/fa69e518-69af-4347-96ff-449488fb37c0)

__Math docs__
![Screenshot_20240929_111916_Chrome](https://github.com/user-attachments/assets/70ecb0be-3dcf-4067-b5ea-6a017d742004)

> As shown above, the current Circle docs have an updated style where **attributes** and **methods** subsection as well as using the new mathematic formatting system. However, it is not the same for other older modules and classes.

## The Aim:
> Discuss the best layout and rewrite the documentation in the new layout (preferrably the circle documentation) to achieve consisgency and parity.

> Feedback is very much wanted.

**Comment by itzpr3d4t0r** [comment]
Yes geometry is a bit unconventional, but this isn't a problem limited to geometry alone as you noted. The real issue is we don't have a standard for docs formatting and docs structure/layout, so many issues arise.

**Comment by MrRedstone058** [comment]
perhaps we can agree on something? this issue is open for ideas anyway
